Shared/Model/DB/AppDatabase.swift

```diff
@@ -153,6 +153,10 @@ extension AppDatabase {
         }
     }
     
+    private func fetchCampaign(with id: UUID, using db: Database) throws -> Campaign? {
```

Comment:
Might want to put the `db` parameter first (or swap the other methods) for consistency.

Reply:
Makes sense 👍